model.js

```diff
@@ -67,3 +67,19 @@ LEFT JOIN users ON reviews.owner = users.username`;
     return reviews;
   });
 };
+
+exports.fetchComments = (review_id) => {
+  return db
+    .query(
+      `SELECT * FROM comments
+  WHERE review_id = $1
+  ORDER BY created_at DESC;`,
+      [review_id]
+    )
+    .then(({ rows: comments }) => {
+      if (comments.length === 0) {
+        return Promise.reject({ status: 404, msg: "not found" });
```

Comment:
If comments array is empty, it might just be because there are no comments on that review, it's not necessarily an error - all we really care about here is that the review_id was valid